src/components/app/BottomBox.css

```diff
@@ -42,7 +42,8 @@
   background: url(/res/img/svg/close-dark.svg) no-repeat center / 16px 16px;
 }
 
-.sourceStatusOverlay {
+.CodeLoadingOverlay,
+.SourceCodeErrorOverlay {
```

Comment:
Ah, they were somewhat intentional because I thought I remembered seeing this form elsewhere, but I must have imagined it. I'll change it to camelCase for consistency. Though we do seem to be a bit unsure about whether to use camelCase or kebab-case, I can find recent examples for both.

Reply:
Yeah, we should fix those inconsistencies as well. I mostly see kebab-case on class names that are related to photon like `photon-button`. Maybe they were imported directly from photon?
But there are also class names that are not tied to photon too.

Reply:
My (non explicit) styleguide was that I used kebab-case for project-transverse styles (such as photon but not only), and camelCase (like it was before) for component-specific styles. TBH I would prefer with the uppercase first letters for these ones because then it's more obvious they're tied to a component.